content/en/docs/tasks/inject-data-application/define-command-argument-container.md

```diff
@@ -1,66 +1,68 @@
 ---
-title: Define a Command and Arguments for a Container
-content_template: templates/task
```

Comment:
The front-matter needs to be reverted to reflect the red lines 2-3 instead of the green lines. This is due to the Hugo migration.